geometry_tutorials/CHANGELOG.rst

```diff
@@ -2,21 +2,7 @@
 Changelog for package geometry_tutorials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-0.2.3 (2020-04-02)
```

Comment:
you can remove this file. this will be automatically generated when we release the package.